specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/storage.json

```diff
@@ -1401,7 +1401,8 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+            "None"
```

Comment:
I believe this will be a breaking change to generated client libraries. 

Will the service ever send back the value "None" in a response? Or will it omit the `"type": "None"` key/value in responses? If it ever sends back `"None"`, it is also a REST API breaking change.

Reply:
It will return the type in the response yes. Considering that this is a breaking change, I will include this in the next API version instead

Reply:
`"None"` is only intended to be a sentinel value for updates, is that correct? If so, it would be better/less breaking if it omitted the key/value if the value is set to `"None"`. This would preserve the current behavior.